src/main/java/org/mongojack/MapReduce.java

```diff
@@ -250,11 +250,6 @@ private MapReduceCommand(String map, String reduce,
                 command.setScope(scope);
             }
             command.setVerbose(verbose);
-            if (extra != null) {
```

Comment:
com.mongodb.MapReduceCommand.addExtraOption() is removed in 3.0.0. was deprecated in 2.13.x